src/test/suite/fixtures/recorded/compoundTargets/bringAirToAfterBatVerticalPastFine.yml

```diff
@@ -1,5 +1,5 @@
-spokenForm: bring air to after column bat past fine
-languageId: markdown
+spokenForm: bring air to after bat vertical past fine
+languageId: typescript
```

Comment:
Deni think we should do a cleanup because all tests I have added are probably on typescript and the majority of them don't require a language server. All take pre post clean actions, testing ranges and more. Should we do a cleanup pr maybe?